packages/ts/lit-form/src/Field.ts

```diff
@@ -120,7 +120,7 @@ export abstract class AbstractFieldStrategy<T = any, E extends FieldElement<T> =
       return true;
     }
 
-    const valid = this.#element.checkValidity();
+    const valid = this.elementValidity();
```

Comment:
Let us try to avoid introducing a method, see suggestions below

```suggestion
    const valid = this.#element.checkValidity();
```